### PR TITLE
Crud course subject task

### PR DESCRIPTION
**Pull request by thedtv-1961**
Redmine: https://edu-redmine.sun-asterisk.vn/issues/14397

Rubocop:
Screenshot from 2019-07-05 08-50-47

Screenshot:
1. Task:
Screenshot from 2019-07-05 08-53-03
Screenshot from 2019-07-05 08-53-25
Screenshot from 2019-07-05 08-53-46

2. Subject:
Screenshot from 2019-07-05 08-54-35
Screenshot from 2019-07-05 08-54-46
Screenshot from 2019-07-05 08-54-59
Screenshot from 2019-07-05 08-55-10

3. Course:
Screenshot from 2019-07-05 08-59-39
Screenshot from 2019-07-05 08-59-47
Screenshot from 2019-07-05 09-00-00
Screenshot from 2019-07-05 09-00-08

### CODE REVIEW

File: app/assets/javascripts/user.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      $('#myModal').modal('hide');
+      $('#showmsg').append('<div class="alert alert-'
+        + data.cls + '">'+data.msg+'</div>');
+    }
+  });
+});
+
+function load_modal(id) {
+  $('#btn-delete').attr('data-id', id)
+}
```
Comment (ghost) [inline app/assets/javascripts/user.js]: tạo hàm xử lý chung cho 4 cái ajax trên

**Review by ghost — COMMENTED** [review]
reviewed by Thuong

File: app/assets/javascripts/user.js (hunk 1 above)
Comment (thedtv-1961) [reply app/assets/javascripts/user.js]: done

**Comment by thedtv-1961** [comment]
ready

**Review by dothidiemthao — COMMENTED** [review]
Pull này chờ pull trước merged rồi hẵng gửi cho trainer nha em, quá nhiều file change review ko nổi (facepalm)

**Pull request closed without merging**